metadata-ingestion/tests/unit/sdk_v2/test_search_client.py

```diff
@@ -175,6 +184,39 @@ def test_unsupported_not() -> None:
         F.not_(env_filter)
 
 
+_default_soft_deleted_filter = {
```

Comment:
minor picky comment

The name suggests that the default value is soft-deleted. I would rename this as `_default_status_filter` (referencing the aspect it applies to) or `_default_not_soft_deleted_filter` (referencing the value itself)

Reply:
fixed